emojivoto-web-app/js/setup_dev_env.sh

```diff
@@ -234,9 +235,9 @@ has_cli
 set_os_arch
 check_init_config
 install_upgrade_telepresence
-run_dev_container
 connect_to_k8s
 connect_local_dev_env_to_remote
+run_dev_container
```

Comment:
If we will do this, we need to update the calls to `display_step` so everything is shown in the right order.